aten/src/ATen/native/cuda/ForeachBinaryOpScalarList.cu

```diff
@@ -6,8 +6,9 @@ namespace at { namespace native {
 
 template<template<class> class Op>
 std::vector<Tensor> foreach_binary_op(TensorList tensors, at::ArrayRef<double> scalars) {
-    std::vector<std::vector<at::Tensor>> tensor_lists; 
+    std::vector<std::vector<at::Tensor>> tensor_lists;
     std::vector<at::Tensor> vec_res;
+    vec_res.reserve(tensors.size());
```

Comment:
Should we do the same for `tensor_lists`? Here and everywhere else

Reply:
Already done, check the other diffs :)